setup.cfg

```diff
@@ -5,3 +5,10 @@ tag = True
 
 [bumpversion:file:eox_core/__init__.py]
 
+[tool:pytest]
+DJANGO_SETTINGS_MODULE = eox_core.settings.test
+
+[coverage:run]
+data_file = .coverage
+omit =
+    venv/*
```

Comment:
emtpy line at the end. Why is the linting not failing because of this?

Reply:
I see. Because this is not a .py file....

Reply:
Empty line added.